.travis.yml

```diff
@@ -3,6 +3,7 @@ node_js:
   - '6'
 # Only use grunt-ci for commits pushed to this repo. Fall back to regular test
 # for pull requests (as secure variables won't be exposed there).
+  - '10'
```

Comment:
This is even, wrong, the code was added below a comment that should be above the `script` property...

Reply:
This is not wrong per se ;-)

Reply:
But as you already closed the PR I can not push any change to this PR anymore.